src/Shell.tsx

```diff
@@ -82,6 +94,28 @@ class ShellContainer extends React.Component<Props, State> implements ShellFunct
         }
     }
 
+    debounceCall = debounce(async (queryString: string) => {
+      const replacedQueryString = queryString
+        .normalize("NFKD")
+        .replace(/[^\w]/g, "");
+      const res = await fetch(
+        `https://feedbot-test-demo-honza.azurewebsites.net/webchat/autosuggest/${replacedQueryString}`
```

Comment:
Tady by to chtelo asi nemit URL napevno 😃 jak se to dela u toho mobiliho uploadu? tam se taky nejak URL skladat musi